geoslurp/discover/fesom/fesom.py

```diff
@@ -1,59 +0,0 @@
-# This file is part of geoslurp-tools.
```

Comment:
Has this file been accidentally deleted? You may need to checkout the file from the most up to date master branch in your own branch first.

Reply:
i've just renamed both files them into xxxQuery.py. It might be reasonable to have different functionnalities in different files, as there's another fesom file: extractprofiles.py